### PR TITLE
Use standardized & translateable strings for the photo show page

### LINKED ISSUE #876: Inconsistent: Photo view page now links back to the harvest or garden it is a photo of
Depending on what it is a photo of you get different UI behaviour:

http://staging.growstuff.org/photos/124 This photo depicts lettuce in CloCkWeRX's garden
http://staging.growstuff.org/photos/127 This photo depicts "Example" (Why not CloCkWeRX's Example garden)

### PR DESCRIPTION
**Pull request by maco**
Fixes #876

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -54,6 +54,10 @@ en:
         crop_plantings: "Everyone's %{crop} plantings"
         owner_plantings: "%{owner} plantings"
 
+  photos:
+    show:
+      thing_by: "A %{thing} by %{owner}"
```
Comment (pozorvlak) [inline config/locales/en.yml]: This will be incorrect if `thing` starts with a vowel sound, but let's cross that bridge if we come to it :-)

**Comment by maco** [comment]
Oh grammar >_<

**Comment by pozorvlak** [comment]
At work, I'm the only native English speaker on my team, so my inner
grammar pedant is less well-restrained than normal :-)

**Comment by maco** [comment]
All our models currently start with consonants, so I didn't think of it.
Though there are people who don't pronounce their h's (and people who _do_
pronounce the h in herb)

**Comment by pozorvlak** [comment]
Indeed; I'm a member of the latter group :-)

**Pull request merged**